contents/blog/best-hipaa-compliant-analytics.md

```diff
@@ -0,0 +1,123 @@
+---
+date: 2022-02-17
```

Comment:
```suggestion
date: 2022-02-15
```

Reply:
Just as a point of feedback, for suggestions like this you can def go ahead and commit them!